clients/banking/src/components/CardItemPhysicalDetails.tsx

```diff
@@ -1453,6 +1503,39 @@ export const CardItemPhysicalDetails = ({
                 currentUserHasRights: isCurrentUserCardOwner || canManageAccountMembership,
                 physicalCard,
               })
+                .with(
+                  {
+                    currentUserHasRights: true,
+                    physicalCard: {
+                      statusInfo: {
+                        __typename: P.not(
+                          P.union(
+                            "PhysicalCardCancelingStatusInfo",
+                            "PhysicalCardCanceledStatusInfo",
+                          ),
+                        ),
+                      },
+                      previousPhysicalCards: [{ isExpired: true }],
```

Comment:
this will only match when `previousPhysicalCards` is of length 1, that might not be intended. we want the first item to be expired, right?